debian/additions/mariadb.conf.d/50-server.cnf

```diff
@@ -0,0 +1,116 @@
+#
+# These groups are read by MariaDB server.
+# Use it for options that only the server (but not clients) should see
+
+# this is read by the standalone daemon and embedded servers
+[server]
+
+# this is only for the mysqld standalone daemon
+[mysqld]
+
+#
+# * Basic Settings
+#
+
+user                    = mysql
+pid-file                = /run/mysqld/mysqld.pid
+socket                  = /run/mysqld/mysqld.sock
+#port                   = 3306
+basedir                 = /usr
+datadir                 = /var/lib/mysql
+tmpdir                  = /tmp
+lc-messages-dir         = /usr/share/mysql
+lc-messages             = en_US
+skip-external-locking
```

Comment:
settings above are default, except for pid-file which is the $datadir/${host?}.pid. Any inclusion reason?

skip-name-resolve is a useful one to include, only if there are no host by domain name grants. broken reverse DNS slows down connections considerably.

Reply:
These are the same as in my.cnf before the change. The Debian "upstreaming" part here was splitting the files into separate ones, no much changes in contents.

Added skip-name-resolve with inline comments to convey the point you had here.